### DISCUSSION TITLE
IT last 14 empty strings

### DISCUSSION THREAD

**Discussion by Gilbum** [aclap-dev/video-downloadhelper#1109] [Language & Translation]
[it]
# v9.1.0.42
v9_panel_footer_show_history_tooltip = '''Mostra la cronologia dei download'''
v9_history_page_title = '''La tua cronologia dei download'''
v9_history_button_clear = '''Azzera la cronologia'''
v9_history_button_start_recording = '''Ricorda la cronologia'''
v9_history_button_stop_recording = '''Non memorizzare la mia cronologia'''
v9_history_no_recording_description = '''La tua cronologia dei download non è memorizzata. Vuoi che Video DownloadHelper la memorizzi?'''
v9_history_no_recording_description_safe = '''Non ti preoccupare, tutto rimane nel tuo computer. Abbiamo a cuore la tua privacy.'''
v9_history_no_entries = '''Nessuna voce ancora.'''
v9_history_input_search = '''Cerca'''
v9_settings_history_limit = '''Dimentica la cronologia dei download dopo X giorni'''
v9_date_today = '''oggi'''
v9_date_yesterday = '''ieri'''
v9_date_x_days_ago = '''$1 giorni fa'''
v9_date_long_ago = '''molto tempo fa'''
[it.txt](https://github.com/user-attachments/files/16515751/it.txt)

**Comment by VoloLibero** [comment]
[it.txt](https://github.com/user-attachments/files/16532441/it.txt)
Here the v9.1.0.42 version
